src/AppBundle/Controller/BundleController.php

```diff
@@ -252,6 +262,42 @@ public function getOrderedSearchBundlesAction($searchText, $page = 1, $order = n
     }
 
     /**
+     * @param string $order
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function renderSortOrderBundleForm($order)
+    {
+        $sortOrderBundleForm = $this->formFactory->create(BundleOrderType::class, [
+            'order' => $order,
+        ]);
+
+        return $this->templating->renderResponse(
+            '@ezdesign/form/bundle_sort_order.html.twig',
+            [
+                'sortOrderBundleForm' => $sortOrderBundleForm->createView(),
+            ]
+        );
+    }
+
+    /**
+     * @param string $searchText
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function renderSearchBundleForm($searchText)
+    {
+        $searchBundleForm = $this->formFactory->create(BundleSearchType::class, [
+            'search' => $searchText,
+        ]);
+
+        return $this->templating->renderResponse(
```

Comment:
Response cache should be disabled by using `$response->setPrivate()` to avoid cache issues using ESI